### ISSUE TITLE
TWRP: 'twrp' command points to 404 documentation URL

### ISSUE THREAD

**Issue by draeath** [omnirom/android_bootable_recovery#76]
~ # twrp
    TWRP openrecoveryscript command line tool, TWRP version 3.0.2-0
    
    Allows command line usage of TWRP via openrecoveryscript commands.
    Some common commands include:
      install /path/to/update.zip
      backup BSDC backupname
      restore backupname BSDC
      factoryreset
      wipe cache
      sideload
      set variable value
      get variable
      decrypt password
    
    See more documentation at http://teamw.in/openrecoveryscript

The URL referenced at the end of this help test is invalid, and I can't seem to find where that information has gone (if it's still valid or exists at all)

**Comment by MSF-Jarvis** [comment]
Yes, that link has been down for quite a while. I'll confirm with the core team if we even need to keep this line in the help message.

**Comment by polyzen** [comment]
Tied to TeamWin/Team-Win-Recovery-Project/issues/697.
